vidarr-cli/src/main/java/ca/on/oicr/gsi/vidarr/cli/TestValidatorScript.java

```diff
@@ -31,10 +29,11 @@ created inside Call() in commandTest.java
       else we create a new directory in the default temporary-file directory
       Note: That path is associate with the default FileSystem which is UNIX in our case
        */
-      final var finalDir = (outputDirectory != null) ? Path.of(outputDirectory, date)
-          : Files.createTempDirectory("vidarr-test-script");
-      final var finalCalculateScript = finalDir.resolve(id);
-      final var finalCalculateDir = finalDir.resolve("output");
+      final var tempDir = Files.createTempDirectory("vidarr-test-script");
+      final var finalDir = (outputDirectory != null) ? Path.of(outputDirectory)
+          : tempDir;
+      final var finalCalculateScript = tempDir.resolve(id);
```

Comment:
why does this use the temporary directory now?

Reply:
The script used to generate results is an important artefact to keep with the results.